tedana/reporting/html_report.py

```diff
@@ -138,44 +143,93 @@ def _update_template_bokeh(bokeh_id, info_table, about, prefix, references, boke
     -------
     HTMLReport : an instance of a populated HTML report
     """
-    resource_path = Path(__file__).resolve().parent.joinpath("data", "html")
-
     # Initial carpet plot (default one)
     initial_carpet = f"./figures/{prefix}carpet_optcom.svg"
 
+    # Get the figures directory - it's in the same directory as the references file
+    figures_dir = os.path.join(os.path.dirname(references), "figures")
```

Comment:
I think this is the culprit. You need to use a relative path, not an absolute path. This will also make the reports slightly more portable, as long as folks include the figures directory when they move the HTML file.

Reply:
That's odd. I use Jinja2 in XCP-D with relative paths without issue. Here's what I do: https://github.com/PennLINC/xcp_d/blob/2cda09ebba3187ccaa691b055cbbf579931ae83e/xcp_d/interfaces/execsummary.py#L74-L82

Maybe instead of `./figures` it should be `figures`?